dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -183,7 +183,7 @@ idTokenCacheEnabled=Cache ID tokens
 idTokenCacheEnabled.desc=Enable caching to save ID tokens in the database and in-memory cache.
 
 jwkRotationTime=JWK rotation time
-jwkRotationTime.desc=Amount of time after which a new JWK is generated (minutes). For example, specify 90 minutes as 90 or 90m. Minimum value is 60 minutes.
+jwkRotationTime.desc=Amount of time after which a new JWK is generated (minutes). For example, specify 90 minutes as 90 or 90m. Minimum value is 0 minutes, where 0 minutes disables key rotation.
```

Comment:
The amount of time after which a new JWK is generated in minutes. For example, specify 90 minutes as 90 or 90m. The minimum value of time is 0 minutes, where 0 minutes disables key rotation.